examples/make-deploy/softlayer.makefile

```diff
@@ -0,0 +1,27 @@
+# Copyright (c) Jupyter Development Team.
+# Distributed under the terms of the Modified BSD License.
+
+softlayer-vm: export SOFTLAYER_CPUS?=4
```

Comment:
I just tried creating a SoftLayer VM.  The SOFTLAYER_CPU variable is singular, not plural.